spyder/plugins/ipythonconsole/tests/test_ipythonconsole.py

```diff
@@ -159,6 +159,8 @@ def __getattr__(self, attr):
                               is_cython=is_cython)
     window.setCentralWidget(console)
 
+    if sys.platform == "win32":
```

Comment:
Please change this to `if not sys.platform == 'darwin'` because I think macOS is the only one that displays this segfault problem.